format: HEP brief results enhancements

nikpap opened the pull request:
* Makes search results more compact.

* Removes bold from DOI and e-Print.

* Results border now occupies all vertical space.

Signed-off-by: Nick Papoutsis <admin@nils.gr>

jmartinm commented:
@nikpap It would be useful for this kind of UI PRs to include a screenshot of the modified part so that it can be seen at a glance and also non-technical people can get a chance to see the updates

jmartinm commented on inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl:
```diff
@@ -57,7 +59,7 @@
       {% if  record.get('dois') %}
       <div class="col-md-6 ">
-        <span class="text-left"><b>DOI:</b></span>{{ record_info() }}
+        DOI: {{ record_info() }}
```
For the DOI macro a more meaningful name like `render_doi()` would be good.

jmartinm commented:
:+1: Looks good.

kaplun commented:
Hi! Would be great if you can attach a screenshot each time with this PRs so that it's clear also for those who can't test quickly test how it will look like.

nikpap commented:
Hey @jmartinm, I've updated this PR and changed the macro name, based on your suggestion.

Here's a screenshot of how the search result would look like:

<img width="866" alt="capture d ecran 2015-11-19 a 09 52 55" src="https://cloud.githubusercontent.com/assets/2742143/11266237/660eee1a-8ea3-11e5-9718-bf4e8474de92.png">

annetteholtkamp commented:
Hi Nick,

the comma after the year in the journal pubnote looks a bit weird, I'd remove it.

Ciao - Annette

On 19 Nov 2015, at 09:54, Nick Papoutsis <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey @jmartinm<https://github.com/jmartinm>, I've updated this PR and changed the macro name, based on your suggestion.

Here's a screenshot of how the search result would look like:

[capture d ecran 2015-11-19 a 09 52 55]<https://cloud.githubusercontent.com/assets/2742143/11266237/660eee1a-8ea3-11e5-9718-bf4e8474de92.png>

—
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/526#issuecomment-157992904>.

nikpap commented:
Updated to remove bold and comma from journal.

jmartinm commented:
Merged 215c0019a1224e3d5de0e0570dce7b7f478bddf6
